scripts/matrix_yttbar.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+
+"""
+A Python script that takes a Grid binned in Rapidity and perform the following operations:
+    - extract the negative rapidity bins and divide the observable predictions by 2
+    - extract the positive rapidity bins and divide the observable predictions by 2
+    - re-order the negative rapidity bin predictions to conform with the commondata
+    - merge the two Grids
+
+To use, check `python ./matrix_yttbar.py --help`, or directly run:
+    `python ./matrix_yttbar.py -g <path_to_grid> -o <output_name>`
+
+To run the check, use `pytest`:
+    `pytest ./matrix_yttbar.py`
+"""
+import numpy as np
+import pineappl
+import rich_click as click
+
+
+class InvalidPineAPPL(Exception):
+    """Check whether the PineAPPL version is valid."""
+
+    pass
+
+
+class TestReverseBins:
```

Comment:
Don't add it here - regardless if it will (already) become a proper pinefarm module or no. This lives just as well in the place where it belongs: `tests/`

Reply:
Moving the tests part into `tests/` is indeed a good idea regardless. Right now I don't have any time at all but will do so as soon as I have some.